geode-core/src/main/java/org/apache/geode/internal/monitoring/executor/SuspendableExecutor.java

```diff
@@ -36,6 +36,11 @@ public void resumeMonitoring() {
     suspended = false;
   }
 
+  @Override
+  public void reportProgress() {
+    setStartTime(System.currentTimeMillis());
```

Comment:
this setStartTime should just be passed 0. SuspendableExecutor has a feature that the thread being monitored can just set its time to 0. When the monitor sees it as 0 will then set the actual time and see if it remains. This helps performance by not having the thread being monitored doing a bunch of System.currentTimeMillis calls which are rather expensive.

Reply:
Thanks for comment. Updated.